scripts/validation/validate_all_environments.py

```
@@ -0,0 +1,839 @@
+                # Remove quotes if present
+                if value.startswith('"') and value.endswith('"'):
+                    value = value[1:-1]
+                elif value.startswith("'") and value.endswith("'"):
+                    value = value[1:-1]
+
+                env_vars[key] = value
+
+    return env_vars
+
+
+def validate_environment(env_path: Path, strict: bool = False) -> EnvironmentValidationReport:
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)